package.json

```diff
@@ -14,7 +14,11 @@
 		"node": ">=12"
 	},
 	"scripts": {
-		"test": "xo && nyc ava && markdownlint '**/*.md'",
+		"lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
+		"lint:js": "xo",
+		"lint:md": "markdownlint '**/*.md'",
```

Comment:
I believe only double quote is supported on Windows.

Reply:
Fixed.